torchinductor/lowering.py

```diff
@@ -846,6 +964,46 @@ def _new_constant(
 register_lowering(aten.new_ones)(new_constant(1))
 
 
+@register_lowering(aten.empty_strided)
+def empty_strided(
+    size, stride, *, dtype=None, layout=None, device=None, pin_memory=None
+):
+    assert isinstance(size, (list, type))
+    assert isinstance(stride, (list, type))
+    assert not pin_memory
+    assert not layout or layout == torch.strided
+    dtype = decode_dtype(dtype) or torch.get_default_dtype()
+    device = device or torch.tensor(0.0).device
+    pointwise = _full(fill_value=0, device=device, dtype=dtype, size=size)
```

Comment:
filling with 0's is typically extra overhead that's not needed for `empty`

Reply:
Agreed.  Inductor doesn't support uninitialized memory, though I think this gets optimized away anyway.